content/en/blog/2023-08-28-announcing-flux-v2.1.0/index.md

```diff
@@ -0,0 +1,177 @@
+  annotation on the resource manifest. The `IfNotPresent` policy is useful to have Flux create an object
+  that will later be managed by another controller.
+- A new flag `--concurrent-ssa` has been introduced in the kustomize-controller to set the number of concurrent
+ server-side operations that will be performed by the controller per object.
+
+You can take a look at the [changelog](https://github.com/fluxcd/flux2/releases/tag/v2.1.0) for the full list of changes.
+
+To upgrade Flux from v0.x to v2.1.0 please follow the
+[Flux GA upgrade procedure](https://github.com/fluxcd/flux2/releases/tag/v2.0.0#upgrade).
+
+To Upgrade Flux from v2.0.x to v2.1.0 either by [rerunning bootstrap](https://fluxcd.io/flux/installation/#bootstrap-upgrade)
+or by using the [Flux GitHub Action](https://github.com/fluxcd/flux2/tree/main/action).
+
+❣️Big thanks to all the Flux contributors who helped us with this release!
+
+### Flagger 1.32.0
+
+This release adds support for suspending a Canary using .spec.suspend. It also fixes a bug where the target
+deployment gets stuck at 0 replicas after the Canary has been deleted. Furthermore, the Canary API has been
+modified to allow specifying the HTTPRoute port using .service.gatewayRefs[].port.
+
+Please see the [changelog](https://github.com/fluxcd/flagger/blob/main/CHANGELOG.md#1310) for the full changes.
+
+## Flux Ecosystem
+
+### Terraform-controller
+
+The ecosystem is buzzing with news about the licensing changes to Hashicorp’s open-source projects
+including Terraform. Weaveworks has released a 
+[statement](https://www.weave.works/blog/statement-for-terraform-hashicorp-license-changes) 
+on this and the impact on the tf-controller.
+
+### VS Code GitOps Extension
+
+Significant performance upgrades and code refactoring has been introduced with VS Code GitOps Tools
+extension version 0.25.0. Previously cluster metadata was loaded using `kubectl get` commands.
+Now, a new javascript client is also used which permits faster loading and real-time watching of cluster resources. 
+`kubectl proxy` is executed in the background for the new client. Rendering of resource treeviews has been reworked
+to minimise data reloading, to maintain collapsible state and to allow visualising resource errors grouped
+by namespaces. Timeout settings were added and bad cluster connections should no longer slow down Clusters treeview rendering.
+
+UI refinements and bug fixes for the new client are ongoing. The most up-to-date UI features can be previewed by
+selecting “Install Pre-Release Version” in the VS Code Extension Browser.
+
+## News from Flux users & the Community!
+
+_If you have not already done so, [use the instructions here](/adopters/)
+or give us a ping and we will help to add you. Not only
+is it great for us to get to know and welcome you to our community. It
+also gives the team a big boost in morale to know where in the world
+Flux is used everywhere._
+
```

Comment:
Let's add a `New Adopters` section with all adopters since 2.0.

Reply:
Thanks for catching that!